### ISSUE TITLE
Example ghe-config command to exempt users from rate limiting has a typo.

### ISSUE THREAD

**Issue by GalaxyAllie** [github/docs#1280]
### What article on docs.github.com is affected?

[Command-line utilities](https://docs.github.com/en/enterprise-server@2.22/admin/configuration/command-line-utilities#ghe-config)

### What part(s) of the article would you like to see updated?

The following example:

```
$ ghe-config app.github.rate_limiting_exempt_users "hubot github-actions"
# Exempts the users hubot and github-actions from rate limits
```

In this example, the correct key has hyphens, not underscores. Therefore it should read:

```
$ ghe-config app.github.rate-limiting-exempt-users "hubot github-actions"
# Exempts the users hubot and github-actions from rate limits
```

### Additional information

I've opened PR #1279 for this.

**Comment by jh8787** [comment]
I

**Comment by janiceilene** [comment]
Thanks for opening a PR @GalaxyAllie! I'll get this triaged for review ✨

**Comment by timchepeleff** [comment]
Just wanted to drop a note that I ran into an issue here. PR contained the solution - would love to see this get merged/reflected in the official docs!

**Comment by janiceilene** [comment]
Thanks for letting us know @timchepeleff! Looks like the PR is waiting on a review and then we'll get it fixed up ⚡

**Comment by shati-patel** [comment]
Fixed by #1279  - the change will be live shortly 😃
